### ISSUE TITLE
Protein coverage calculation

### ISSUE THREAD

**Issue by apsteinberg** [Nesvilab/FragPipe#1810]
Hi all,

I had a quick question regarding how the calculation of protein coverage is performed in the `protein.tsv` report. I was trying to compute protein coverage by using the results that are in the `peptide.tsv` -- basically, I was taking the `Protein Start` and` Protein End` positions listed for each peptide related to a given protein in the peptide.tsv file, and using these to compute protein coverage. I am finding there is a slight discrepancy between the values. Does the protein coverage listed in the `protein.tsv` report include some additional peptides? 

Thanks,
Asher

**Comment by fcyu** [comment]
Hi Asher,

It was calculated by ProteinProphet. If I remember it correctly, it uses all peptides from all samples/input files.

Best,

Fengchao

**Comment by anesvi** [comment]
I though we were even going to remove that column, Fengchao? Yes I think it is from ProteinProphet which uses peptides with a weight of 0.5 and above. But philosopher is using a modified razor assignment logic so the set of peptides assigned as razor peptides to each protein is different. But I have to check to see what we are really printing in that column.

**Comment by apsteinberg** [comment]
Hi Fengchao and Alexey,

Thanks for the quick response. If I am understanding correctly, the peptides in the `peptide.tsv` are slightly different than those used for the protein coverage calculation that is included in the `protein.tsv` output? 

Best,
Asher

**Comment by fcyu** [comment]
That's correct. The protein coverage calculation was not well implemented in Philosopher. We need to change that.

Best,

Fengchao

**Comment by apsteinberg** [comment]
Got it. Thank you both, this is super helpful!

Best,
Asher
